### ISSUE TITLE
Scrolling glitch

### ISSUE THREAD

**Issue by sochalewski** [danielgindi/ios-charts#873]
I asked about it in already closed issue #430, but no answer there.

So I've got a line chart view and an issue while scrolling. I noticed that it's because of `drawFilledEnabled` set to `true`. Any idea how to fix it?

Here's my setup:
```swift
chartView.scaleYEnabled = false
chartView.scaleXEnabled = false
chartView.pinchZoomEnabled = false
chartView.doubleTapToZoomEnabled = false
chartView.xAxis.drawGridLinesEnabled = false
chartView.leftAxis.customAxisMin = 0.0
chartView.autoScaleMinMaxEnabled = false
chartView.leftAxis.drawGridLinesEnabled = false
chartView.rightAxis.enabled = false
chartView.legend.form = .Circle
```

And here is screencast showing what's the matter: https://dl.dropboxusercontent.com/u/5765015/glitch.mov

**Comment by liuxuan30** [comment]
![glitch](https://cloud.githubusercontent.com/assets/4375169/13981338/89b6326e-f11e-11e5-910e-2bec7bcfdffa.png)
Just get a screenshot when the isue is happening like above.
It seems it cannot fetch the left most point out of chart, so the filling is broken, as well as the line. 

What version of the library are you using? Have you tried the latest code? Recently @danielgindi has fixed and merged some related issues. I tried with ChartsDemo, it seems good. Please check.

**Comment by sochalewski** [comment]
You are right, the demo works just fine.

I use recent 2.2.3 via CocoaPods, but it looks like there are changes merged after 2.2.3.
When I specify `:git` in my Podfile forcing recent version from GitHub to be used everything seems to be fine.
Could you please update release issued via CocoaPods, please?

Thank you for your help, @liuxuan30. Love this library :)
